app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationViewModel.kt

```diff
@@ -31,6 +31,9 @@ class ConversationViewModel(
     private val storage: Storage
 ) : ViewModel() {
 
+    val showSendAfterApprovalText: Boolean
+        get() = recipient?.run { address.isGroup.not() && hasApprovedMe().not() } ?: false
```

Comment:
I think `Recipient` also has a `isContactRecipient` (short-hand for `address.isContact()`) which we could use here instead of `address.isGroup.not()`

Reply:
done